app/src/poi.js

```diff
@@ -31,6 +31,42 @@ export default function Poi() {
         },
       });
 
+      const popup = new mapboxgl.Popup({
+        closeButton: false,
+        closeOnClick: false,
+      });
+      map.current.on("mousemove", "points-of-interest", (e) => {
+        // Change the cursor to a pointer when the mouse is move over the points-of-interest layer.
+        map.current.getCanvas().style.cursor = "pointer";
+        // Copy coordinates array.
+        const coordinates = e.features[0].geometry.coordinates.slice();
+        const name = e.features[0].properties.business_name;
+        const address =
+          e.features[0].properties.address +
+          ", " +
+          e.features[0].properties.city +
+          ", " +
+          e.features[0].properties.state +
+          ", " +
+          e.features[0].properties.zip +
+          ", " +
+          e.features[0].properties.country;
+        const category =
+          e.features[0].properties.category_name +
+          " (" +
+          e.features[0].properties.category_id +
+          ") ";
```

Comment:
This can be improved by using string formatting in JS. Check the usage of `{}` in strings in JS.